integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/MessageSender.java

```diff
@@ -110,6 +114,38 @@ public PutMessageResponseType sendMessage(EDUCore message) {
         return createOkResponse();
     }
 
+    public void sendMessage(ConversationResource conversation) throws MessageContextException, IOException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: no.difi.meldingsutveksling.noarkexchange.MessageContextException, java.io.IOException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)